components/Search/FuseSearchRecipe.js

```diff
@@ -1,15 +1,21 @@
 import AllRecipesList from "../AllRecipesList";
 import { useState, useEffect } from "react";
```

Comment:
maybe u should remove the useEffect import here

Reply:
useEffect is used in this file, so we need it ;)